deployment/core-infrastructure/kustomize-templates/gateway/kustomization.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+metadata:
+  name: maas-gateway
+
+resources:
+- model-routing.yaml
+- openshift-routes.yaml
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)